source/common/config/filesystem_subscription_impl.h

```diff
@@ -33,8 +36,10 @@ class FilesystemSubscriptionImpl : public Config::Subscription<ResourceType> {
     callbacks_ = &callbacks;
     watcher_->addWatch(path_, Filesystem::Watcher::Events::MovedTo, [this](uint32_t events) {
       UNREFERENCED_PARAMETER(events);
-      refresh();
+      refresh(false);
     });
+    // Attempt to read, gracefully fail if the file isn't there yet.
```

Comment:
Sure that makes sense, but in this particular case for the filesystem impl why have the ignore error case on initial startup?